### PR TITLE
Docs: fix dead links

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

### CODE REVIEW

File: content/en/docs/contribution.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ Please make sure to read and observe our [Code of Conduct](https://github.com/vo
...
 The goal of the community is to develop a volcano system which is useful for runnning high performance workloads such as AI,ML,Deep Learning Application on top of Kubernetes. To build a such volcano system at such scale requires the support of a community with similar aspirations.
 
-- See [Community Membership](https://github.com/volcano-sh/website/blob/master/content/en/docs/community-membership.md) for a list of various community roles. With gradual contributions, one can move up in the chain.
+- See [Community Membership](https://github.com/volcano-sh/volcano/blob/master/community-membership.md) for a list of various community roles. With gradual contributions, one can move up in the chain.
 
 
 # Getting started
 
-- Read the [setup](https://github.com/volcano-sh/website/blob/master/content/en/docs/deployment.md) for build/deploy instructions.
+- Read the [setup](./installation.md) for build/deploy instructions.
```
Comment (Thor-wl) [inline content/en/docs/contribution.md:53]: please keep original link. This link is invalid.

File: content/zh/docs/contribution.md [hunk 2]
```diff
@@ -47,12 +47,12 @@ Volcano 是一个社区驱动的开源项目，致力于打造健康、友好和
...
 区贡献者的支持。
 
-- 请查看 [社区角色](https://github.com/volcano-sh/website/blob/master/content/en/docs/community-membership.md) 了解社区角色的
+- 请查看 [社区角色](https://github.com/volcano-sh/volcano/blob/master/community-membership.md) 了解社区角色的
   具体情况。随着贡献度的提高，您的角色会逐渐升级。
 
 ## 开始
 
-- 查看[安装文档](https://github.com/volcano-sh/website/blob/master/content/en/docs/deployment.md) 了解更多关于编译和部署的细节.
+- 查看[安装文档](./installation.md) 了解更多关于编译和部署的细节.
```
Comment (Thor-wl) [inline content/zh/docs/contribution.md:55]: same here

File: content/en/docs/contribution.md (hunk 1 above)
Comment (daixiang0) [reply content/en/docs/contribution.md:53]: The original page is 404.

Comment (daixiang0) [reply content/en/docs/contribution.md:53]: I think it is a relative path problem.

**Pull request merged**